sdk/cognitiveservices/cognitiveservices-luis-authoring/src/lUISAuthoringClient.ts

```diff
@@ -1,8 +1,4 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
```

Comment:
Copyright should not be deleted.
@sarangan12 Please share the flag to use so that the copyright is included